ci-operator/step-registry/network/conf/ingress-avoid-worker/network-conf-ingress-avoid-worker-commands.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-avoid-workers-99-config.yaml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 50-worker-fix-ipi-rwn
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/kubernetes/manifests/keepalived.yaml
+          mode: 0644
+          contents:
+            source: data:,
+EOF
+
+cat >> "${SHARED_DIR}/manifest_cluster-ingress-default-ingresscontroller.yaml" << EOF
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  nodePlacement:
+    nodeSelector:
+      matchLabels:
+        node-role.kubernetes.io/master: ""
```

Comment:
The PR https://github.com/openshift/cluster-ingress-operator/pull/858 enabled ingress operator can use node affinity rule to router deployement that will not allow router to run on remote worker node (node with custom role label), and it has been implemented in 4.13.
From my understanding, supporting remote worker doesn't mean to make ingress router pod running on master only, because routers are always running on normal worker node. Maybe we should create post action step to update the node label and ingresscontroller's `spec.nodePlacement`.  see also https://issues.redhat.com/browse/OCPQE-13223

Reply:
I would suggest we follow doc [https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configure[…]n-workflow](https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configure-network-components-to-run-on-the-control-plane_ipi-install-installation-workflow)

Reply:
@lihongan I guess remote worker PR https://github.com/openshift/cluster-ingress-operator/pull/858 only on IPI BM now see https://github.com/openshift/baremetal-runtimecfg/pull/224/files, So on vsphere platform, the remote worker do not have the label `node.openshift.io/remote-worker`

Reply:
@zhaozhanqi ingress PR (node affinity rule) is working for all platforms, but just like you said it depends on the labels if added to remote worker or not. If vSphere doesn't do that, then the ingress PR cannot work. 
Seems you're testing https://issues.redhat.com/browse/OPNET-27, do you know if vSphere will add the label to remote worker?

Reply:
If that, we can continue the original solution to test remote workers on vsphere.

Reply:
BTW, actually per https://github.com/openshift/cluster-ingress-operator/pull/858/files, once it was merged, it will  applicable for ingress router, why it is bound with a specific platform? Ingress router will have different behavior for the remote workers on different platforms.

Reply:
Ingress is the consumer of the label `node.openshift.io/remote-worker` , I think we can add the label manually if no other operator/component to revert it.